src/Telegram.Bot/Types/ChatMember.cs

```diff
@@ -20,5 +21,94 @@ public class ChatMember
         /// </summary>
         [JsonProperty("status", Required = Required.Always)]
         public ChatMemberStatus Status { get; set; }
+
+        /// <summary>
+        /// Optional. Restictred and kicked only. Date when restrictions will be lifted for this user, unix time
+        /// </summary>
+        [JsonProperty(PropertyName = "until_date", Required = Required.Default)]
+        private long UntilDateUnix = 0;
+
+        /// <summary>
+        /// Optional. Restictred and kicked only. Date when restrictions will be lifted for this user, UTC time
+        /// </summary>
+        public DateTime UntilDate { get { return DateTime.FromFileTimeUtc(UntilDateUnix); } set { UntilDateUnix = value.ToFileTimeUtc(); } }
```

Comment:
this can be done without a private property: https://github.com/MrRoundRobin/telegram.bot/blob/a55f7dd66e0fa2be0edb5adeedbaa93ba60d31e9/src/Telegram.Bot/Types/Message.cs#L33

`[JsonConverter(typeof (UnixDateTimeConverter))]`